packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/semanticdata/services/SemanticDataUpdateService.java

```diff
@@ -45,4 +52,38 @@ public void updateDocuments(ICause cause, AggregateReference<SemanticData, UUID>
             this.semanticDataRepository.save(semanticData);
         });
     }
+
+    @Override
+    public IResult<Void> addDependency(ICause cause, UUID semanticDataId, UUID dependencySemanticDataId) {
```

Comment:
Since we have merged library-to-library dependencies and project-to-library dependencies into semantic data dependencies, we now need to provide update capabilities to be able to add dependencies to a project (e.g. import a library).
This is dangerous, because it also allows to update the dependencies of a library, although libraries should be immutable. I don't know where we should ensure this.